File: frontend/src/app/core/models/ContenitoreFormNuovaRichiestaNoleggio.ts
import {VeicoloPrenotazioneDTO} from '@core/models/VeicoloPrenotazioneDTO';

export interface ContenitoreFormNuovaRichiestaNoleggio {
  veicolo: VeicoloPrenotazioneDTO,
  dataInizio: string,
  dataFine: string,
  oraInizio: string,
  oraFine: string
  motivazione: string
}

File: backend/src/main/java/it/unical/fleetgo/backend/Models/DTO/ContenitoreFormNuovaRichiestaNoleggio.java
package it.unical.fleetgo.backend.Models.DTO;


import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ContenitoreFormNuovaRichiestaNoleggio {
    private VeicoloPrenotazioneDTO veicolo;
    private String dataInizio;
    private String dataFine;
    private String oraInizio;
    private String oraFine;
    private String motivazione;
}
